cmake/emu/qemu.cmake

```diff
@@ -37,8 +37,8 @@ if(CONFIG_QEMU_UEFI_BOOT)
 endif()
 
 set(qemu_targets
-  run
-  debugserver
+  run_qemu
+  debugserver_qemu
```

Comment:
Please remember to update all occurrences of debugserver.
The `debugserver_qemu` no longer waits for a connections, because this line is not updated:
 https://github.com/zephyrproject-rtos/zephyr/blob/f397e9b6c2e5268009e303210807d4b9045e1eb4/cmake/emu/qemu.cmake#L337

Note: there could be more places in need of an update, so please do a thorough search.

Reply:
Thanks for spotting that, fixed. I didn't find any other places that needed to be updated, so I'm resolving this conversation.